frontend/www/js/omegaup/arena/arena.ts

```diff
@@ -166,6 +166,40 @@ function getMaxScore(
   return maxScore;
 }
 
+function findElement(
+  element: HTMLElement | null,
+  itemSelector: string | null,
+): HTMLElement | null {
+  if (element) {
+    if (itemSelector !== null) {
+      element = element.querySelector(itemSelector);
+    }
+  }
+  return element;
```

Comment:
acá se puede usar el guard clause pattern?

```suggestion
  if (!element) {
    return element;
  }
  if (itemSelector === null) {  // también se vale `!element || itemSelector === null`
    return element;
  }
  return element.querySelector(itemSelector);
```